README.md

```diff
@@ -14,340 +14,57 @@ The aim of **Pyccel** is to provide a simple way to generate automatically, para
-
-For Fortran it supports
-
--   [GFortran](https://gcc.gnu.org/fortran/)
--   [Intel® Fortran Compiler](https://software.intel.com/en-us/fortran-compilers)
--   [PGI Fortran](https://www.pgroup.com/index.htm)
-
-For C it supports
-
--   [GCC](https://gcc.gnu.org/)
--   [Intel® Compiler](https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/dpc-compiler.html)
--   [PGI](https://www.pgroup.com/index.htm)
-
-In order to perform fast linear algebra calculations, Pyccel uses the following libraries:
-
--   [BLAS](http://www.netlib.org/blas/)(Basic Linear Algebra Subprograms)
--   [LAPACK](http://www.netlib.org/lapack/)(Linear Algebra PACKage)
-
-Finally, Pyccel supports distributed-memory parallel programming through the Message Passing Interface (MPI) standard; hence it requires an MPI library like
-
--   [Open-MPI](https://www.open-mpi.org/)
--   [MPICH](https://www.mpich.org/)
--   [Intel® MPI Library](https://software.intel.com/en-us/mpi-library)
-
-We recommend using GFortran/GCC and Open-MPI.
-
-Pyccel also depends on several Python3 packages, which are automatically downloaded by pip, the Python Package Installer, during the installation process. In addition to these, unit tests require additional packages which are installed as optional dependencies with pip, while building the documentation requires [Sphinx](http://www.sphinx-doc.org/).
-
-### Linux Debian-Ubuntu-Mint
-
-To install all requirements on a Linux Ubuntu machine, just use APT, the Advanced Package Tool:
-
-```sh
-sudo apt update
-sudo apt install gcc
-sudo apt install gfortran
-sudo apt install libblas-dev liblapack-dev
-sudo apt install libopenmpi-dev openmpi-bin
-sudo apt install libomp-dev libomp5
-```
-
-### Linux Fedora-CentOS-RHEL
-
-Install all requirements using the DNF software package manager:
-
-```sh
-su
-dnf check-update
-dnf install gcc
-dnf install gfortran
-dnf install blas-devel lapack-devel
-dnf install openmpi-devel
-dnf install libgomp
-exit
-```
-
-Similar commands work on Linux openSUSE, just replace `dnf` with `zypper`.
-
-### Mac OS X
-
-On an Apple Macintosh machine we recommend using [Homebrew](https://brew.sh/):
-
-```sh
-brew update
-brew install gcc
-brew install openblas
-brew install lapack
-brew install open-mpi
-brew install libomp
-```
-
-This requires that the Command Line Tools (CLT) for Xcode are installed.
+-   [Quick-start Guide](https://github.com/pyccel/pyccel/blob/devel/docs/quickstart.md)
 
-### Windows
+-   [Installation](https://github.com/pyccel/pyccel/blob/devel/docs/installation.md)
 
-Support for Windows is still experimental, and the installation of all requirements is more cumbersome.
-We recommend using [Chocolatey](https://chocolatey.org/) to speed up the process, and we provide commands that work in a git-bash sh.
-In an Administrator prompt install git-bash (if needed), a Python3 distribution, and a GCC compiler:
+-   [Contributing](https://github.com/pyccel/pyccel/blob/devel/docs/CONTRIBUTING.md)
```

Comment:
I think that "Contributing" also deserves a section, which can use the same approach as the "Installation" section.

Reply:
Fixed [4300001](https://github.com/pyccel/pyccel/pull/1438/commits/4300001043315b0f1954ae8327c8abfad239e3a5)  + [492d13f](https://github.com/pyccel/pyccel/pull/1438/commits/492d13f7f35ec90ff7fe6b41cbf20bc4bfd347a6)